src/Plugin/GraphQL/DataProducer/Entity/EntityQuery.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Drupal\graphql\Plugin\GraphQL\DataProducer\Entity;
+
+use Drupal\Core\Cache\RefinableCacheableDependencyInterface;
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\graphql\Plugin\GraphQL\DataProducer\DataProducerPluginBase;
+use GraphQL\Error\UserError;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Builds and executes Drupal entity query.
+ *
+ * @DataProducer(
+ *   id = "entity_query",
+ *   name = @Translation("Load entities"),
+ *   description = @Translation("Loads entities."),
+ *   produces = @ContextDefinition("entities",
```

Comment:
"entities" is not a typed data type. It actuall produces a list of string. So the context definition here should be a "multiple" (multiple => true) of "string".